src/releng_docs/run.py

```diff
@@ -5,6 +5,7 @@
 
 HERE = os.path.dirname(__file__)
 
+os.system('/run/current-system/sw/bin/make html')
```

Comment:
i think i fixed this in the master already. @lundjordan can you rebase on ``origin/master``?